app/src/main/java/com/example/moodtrackerproject/SplashActivity.kt

```diff
@@ -0,0 +1,14 @@
+package com.example.moodtrackerproject
+
+import android.content.Intent
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+
+class SplashActivity : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        var intent = Intent(this, MainActivity::class.java)
+        startActivity(intent)
```

Comment:
Such code (like starting an activity) relates to navigation/routing. That's why encapsulate this logic in `Router` class (singleton)